src/app/globals.css

```
@@ -63,20 +63,335 @@ body {
+@utility shadow-magma-glow {
   box-shadow: var(--shadow-magma);
 }
 
+/* ==========================================================================
+   Tailwind CSS Component Layer (Consolidated Design Token Primitives)
+   ========================================================================== */
+@layer components {
+  /* 1. Primary Magma CTA Button */
+  .btn-magma {
+    @apply inline-flex items-center justify-center gap-2 font-bold rounded-xl text-white outline-none select-none transition-all duration-200 cursor-pointer;
+    background: var(--gradient-magma);
```

Stylelint (17.14.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)